File: Java/4A-Watermelon.java
/*
	Solution to Codeforces problem 4A
	Copyright (c) GeneralMing. All rights reserved.

	https://github.com/GeneralMing/codeforces
*/

import java.util.*;
public class Watermelon_4A
{
	public static void main(String[] args)
	{
		int w;

		Scanner scanner = new Scanner(System.in);
		w = Integer.parseInt(scanner.nextLine());
		if(w%2 != 0 || w == 2)
			System.out.println("NO");
		else
			System.out.println("YES");

	}
}

File: C/4A-Watermelon.c
/*
	Solution to Codeforces problem 4A
	Copyright (c) GeneralMing. All rights reserved.

	https://github.com/GeneralMing/codeforces
*/

#include<stdio.h>

int main()
{
	int w;
	scanf("%d", &w);
	if(w%2 != 0 || w == 2)
		printf("NO\n");
	else
		printf("YES\n");
	return 0;
}